src/helpers/tests/geoChart/geoChart.spec.ts

```diff
@@ -60,4 +65,14 @@ describe("getGeoData", () => {
             },
         );
     });
+
+    it("calculateAverage should return expected value ", () => {
+        const values: number[] = [1, 2, 3, 4, 5, 6];
+        expect(calculateAverage(values)).toEqual(3.5);
+    });
+
+    it("getFormatFromExecutionResponse should return format of Size Measure ", () => {
+        const executionResponse = getExecutionResponse(true, false, false, true, false);
+        expect(getFormatFromExecutionResponse(0, executionResponse)).toEqual("#,##0");
+    });
```

Comment:
we should follow the convention: tests start with 'should ....'

Reply:
fixed